masq_lib/src/multi_config.rs

```diff
@@ -949,6 +1037,40 @@ pub mod tests {
         assert_eq!(subject.args(), command_line);
     }
 
+    #[test]
+    fn command_line_vcl_return_value_from_vcl_args_by_name() {
+        let command_line: Vec<String> = vec![
+            "",
+            "--first-value",
+            "/nonexistent/directory",
+            "--takes_no_value",
+            "--other_takes_no_value",
+        ]
+        .into_iter()
+        .map(|s| s.to_string())
+        .collect();
+
+        let subject = CommandLineVcl::new(command_line.clone());
+        let existing_value = match subject
```

Comment:
I think this `match` at the beginning could be replaced with a `.map()` at the end. Also, this sequence of code that you write once for `existing_value` and once for `non_existing_value` could probably be made into a local function or closure and called twice. If locating a `VclArg` inside a `VirtualCommandLine` by name turns out to be useful enough, it could turn into a local utility function that other tests could use.